src/capwap_config.erl

```diff
@@ -143,6 +130,8 @@ wtp_init_wlan_keymgmt(WLAN = #wtp_wlan_config{
 
 wtp_init_wlan(_CN, _Radio, {ssid, _}, WLAN) ->
     WLAN;
+wtp_init_wlan(_CN, _Radio, {wlan_id, Value}, WLAN) ->
```

Comment:
is it really necessary to make the wlan_id configurable? The wlan_id is normally generated automatically and I don't really see the need to explicitly set it through the configuration.

Reply:
I have this option in configs:

 * http, like this
         "wlans":  [ {"wlan_id":1, "ssid":"test 1", "suppress_ssid":0}  ]
 * envs, like this
                {wlans, [
                        [{wlan_id, 1}, {ssid, <<"test ate 2">>}, {suppress_ssid, 0}]
                ]}]}

Reply:
I know that you have it in there, but why do you need in there?

Reply:
For successful merging